tests/usecase/test_check_terminate_process_on_crashing_app.py

```diff
@@ -0,0 +1,25 @@
+import basetest
+import subprocess
+
+
+class TestCaseTerminateChildProcessesCompleteOnCrashingApp(basetest.BaseTest):
+
+    def setUp(self):
+        self.setUpCF('Sample-StartError.mpk', env_vars={
+            'DEPLOY_PASSWORD': self.mx_password,
+            'METRICS_INTERVAL': '10',
+        })
+
+    def test_verify_termination_logs(self):
+        try:
+            self.startApp(start_timeout=15)
+            self.fail("App should not have been started")
+        except subprocess.CalledProcessError:
+            pass
+        self.assert_string_in_recent_logs(self.app_name, 'start failed, stopping')
+        self.assert_string_in_recent_logs(self.app_name, 'Stopping M2EE ...')
+        self.assert_string_in_recent_logs(self.app_name, 'SIGTERM sent to all child processes.')
```

Comment:
I think we should only check the actual result of the problem (`health check never passed`) and leave out checks for the extra logging that we added in this PR.